src/web/app/components/simple-modal/simple-modal.component.html

```diff
@@ -37,5 +37,5 @@ <h5 class="modal-title">
           'btn-warning' : type === SimpleModalType.WARNING,
           'btn-info' : type === SimpleModalType.INFO,
           'btn-primary' : type === SimpleModalType.NEUTRAL }"
-          (click)="activeModal.close()">{{ confirmMessage }}</button>
+          (click)="activeModal.close(redirect())">{{ confirmMessage }}</button>
```

Comment:
This is an abuse of the close method. The parameter is used to pass a result to the emitted observable or resolved promise after the function is called, not a way to perform an additional action. 

The modal also ideally should only be concerned with what it has to display and shouldn't have to concern itself with what actions to perform after the modal is closed/dismissed.

The idiomatic way to perform an action after a modal has been closed/dismissed in this context is to subscribe to the the appropriate observable that is emitted after the modal is closed/dismissed or via callbacks when the `result` promise is resolved (see https://ng-bootstrap.github.io/#/components/modal/api#NgbModalRef)

e.g.
```
const modalRef = openModal(...);
modalRef.closed.subscribe(() => {
  // handle navigation here
})
```